Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/Plugins/StandardPlugins/Source/AnimGraph/ParameterEditor/Vector3GizmoParameterEditor.cpp

```diff
@@ -101,31 +106,25 @@ namespace EMStudio
 
         // Setup the translation manipulator
         AzToolsFramework::ConfigureTranslationManipulatorAppearance3d(&m_translationManipulators);
-        m_translationManipulators.InstallLinearManipulatorMouseMoveCallback(
-            [this](const AzToolsFramework::LinearManipulator::Action& action)
-            {
-                OnManipulatorMoved(action.LocalPosition());
-            });
-
-        m_translationManipulators.InstallPlanarManipulatorMouseMoveCallback(
-            [this](const AzToolsFramework::PlanarManipulator::Action& action)
-            {
-                OnManipulatorMoved(action.LocalPosition());
-            });
-
-        m_translationManipulators.InstallSurfaceManipulatorMouseMoveCallback(
-            [this](const AzToolsFramework::SurfaceManipulator::Action& action)
+        auto mouseMoveHandlerFn = [this](const auto& action) {
```

Comment:
Nice refactor here! 🙂

Reply:
I like lambdas like this, allows you to scope things down a level from like a class method. so you can't use it in another context :P.